src/melee/lb/lbcommand.c

```diff
@@ -77,7 +77,7 @@ void Command_07(CommandInfo* info)
 void Command_08(CommandInfo* info)
 {
     info->u.data_position += 1;
-    info->timer = FLT_MAX;
+    info->timer = F32_MAX;
```

Comment:
```suggestion
    info->timer = FLT_MAX;
```
FLT_MAX is in the C standard

Reply:
I hear you, but we're in a weird spot because we need ports to have the same behavior and `FLT_MAX` does not guarantee a particular value (it happens to be the 32-bit IEEE max on most platforms). I will revert it if you insist but I just want to guarantee that this value is what HAL used, without relying on literals.

Reply:
If you want to specify f32, than the header that does `typedef float f32` can also include <float.h> and `#define F32_MAX FLT_MAX`.

Reply:
But wouldn't that make `F32_MAX` dependent on the platform? I'm not sure I'm understanding.

Reply:
Yes, and platform-dependent typedefs are in Runtime/platform.h. That's where you've typedef'd f32 as float. So Runtime/platform.h "knows" that a float is 32 bits, and it can also include <float.h> and define `F32_MAX` to be equivalent to `FLT_MAX`. And then if you want to use F32_MAX that definition is available in Runtime/platform.h.

Reply:
`typedef float f32` is technically an implementation detail on `MWERKS_GEKKO` but as far as I understand, `float.h` doesn't guarantee that `FLT_MAX` refers to 32-bit floats, correct?

This is my understanding:
- `f32`: Guaranteed 32-bits on any platform
- `F32_MAX`: Guaranteed to be `3.4028235E38` (bit for bit)
- `FLT_MAX`: Guaranteed to be the maximum single-precision floating-point value for the current platform

... sorry if I'm being dumb.

Reply:
Ok yeah, we're on the same page. I was thinking this header should reuse the knowledge that f32 == float for the F32/FLT define. TBH, it's not that important, we can leave it hardcoded here for now. I'm going to resolve this thread.